db/default/zones.rb

```diff
@@ -1,22 +1,24 @@
 # frozen_string_literal: true
 
 unless Spree::Zone.find_by(name: "EU_VAT")
-  eu_vat = Spree::Zone.create!(name: "EU_VAT",
-                               description: "Countries that make up the EU VAT zone.")
+  eu_vat = Spree::Zone.create(name: "EU_VAT",
+                              description: "Countries that make up the EU VAT zone.")
 
   ["Poland", "Finland", "Portugal", "Romania", "Germany", "France",
    "Slovakia", "Hungary", "Slovenia", "Ireland", "Austria", "Spain",
    "Italy", "Belgium", "Sweden", "Latvia", "Bulgaria", "United Kingdom",
    "Lithuania", "Cyprus", "Luxembourg", "Malta", "Denmark", "Netherlands",
    "Estonia"].each do |name|
-    eu_vat.zone_members.create!(zoneable: Spree::Country.find_by!(name:))
+    eu_vat.zone_members << Spree::ZoneMember.create(zoneable: Spree::Country.find_by!(name:))
   end
+  eu_vat.save!
```

Comment:
Would it be better to use `new` here?

Otherwise you create a zone member in the database with a `nil` zone_id and when you call `eu_vat.save!` all the zone members get updated to store the zone_id on them.

Reply:
Indeed that's better, thanks !